packages/run-protocol/test/vaultFactory/test-vaultFactory.js

```diff
@@ -635,7 +635,8 @@ test('price drop', async t => {
   const finalNotification = await E(vaultNotifier).getUpdateSince();
   t.is(finalNotification.value.vaultState, Phase.LIQUIDATED);
   t.deepEqual(finalNotification.value.locked, aeth.make(2n));
-  t.is(debtAmountAfter.value, 30n);
```

Comment:
We should add a check that the shortfall got recorded in the total shortfall on the vaultmanager, or some such.

Reply:
This was happening a few lines down so I just moved it up. Same for the other one.